internal/cmd/server.go

```diff
@@ -137,9 +137,10 @@ The kernel is used to run long running processes like shells and interacting wit
 	cmd.Flags().StringVarP(&addr, "address", "a", defaultAddr, "Address to create unix (unix:///path/to/socket) or IP socket (localhost:7890)")
 	cmd.Flags().BoolVar(&devMode, "dev", false, "Enable development mode")
 	cmd.Flags().BoolVar(&enableRunner, "runner", true, "Enable runner service (legacy, defaults to true)")
-	cmd.Flags().BoolVar(&enableAILogs, "ai-logs", false, "Enable logs to support training an AI")
+	// The AIFlag is no longer used, we can remove it as soon as the option has been removed from the frontend.
+	cmd.Flags().BoolVar(&enableAILogs, "ai-logs", false, "(Obsolete) This flag is no longer used.")
```

Comment:
Perhaps add `_ = cmd.Flags().MarkDeprecated("ai-logs", "This flag is no longer used.")` instead of a note?

Reply:
Done.